src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_blob_scenarios.py

```diff
@@ -95,6 +95,13 @@ def verify_blob_upload_and_download(self, group, account, file_size_kb, blob_typ
             [JMESPathCheck('properties.contentSettings.contentType', 'application/test-content'),
              JMESPathCheck('properties.contentLength', file_size_kb * 1024)])
 
+        # check that blob properties can be set back to null
+        self.storage_cmd('storage blob update -n {} -c {} --content-type ""',
```

Comment:
It may not be relevant here, but in general it is important to be able to distinguish between unsetting and setting to the empty.

Does the CLI have any existing story for drawing this distinction?